File: Day-48/MoveZeros.py
def MoveZeros(arr):
    point = 0
    for i in range(0, len(arr)):
        if(arr[i]!=0):
            temp = arr[i]
            arr[i] = arr[point]
            arr[point] = temp
            point += 1
    print(arr)        

MoveZeros([0, 1, 0, 3, 12])

File: Day-48/MoveZeros.js
function MoveZeros(arr) {
  let point = 0;
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] !== 0) {
      let temp = arr[i];
      arr[i] = arr[point];
      arr[point] = temp;
      point++;
    }
  }

  console.log(arr);
}

MoveZeros([0, 1, 0, 3, 12]);
